proto/project.proto

```diff
@@ -141,6 +141,12 @@ message Project {
       Paper project_paper = 1;
       google.protobuf.FieldMask mask = 2;
     }
+
+    message Next {
+      string project_id = 1;
+      string paper_id = 2;
+      bool review_mode_active = 3;
```

Comment:
I think this should be replaced with extra rpc calls.
Something like:
```
GetNextPaperToReview
GetNextPaper
```
depending on the action taken for each value of the boolean.

What's your opinion on this? I am open to discussion.